docs/resources/dataarts_studio_permission_set.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "DataArts Studio"
+---
+
+# huaweicloud_dataarts_studio_permission_set
+
+Manages DataArts Studio permission set resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+variable "name" {}
+variable "manager_id" {}
+variable "manager_name" {}
+variable "manager_type" {}
+
+resource "huaweicloud_dataarts_studio_permission_set" "test" {
+  workspace_id = var.workspace_id
+  name         = var.name
+  parent_id    = "0"
+  manager_id   = var.manager_id
+  manager_name = var.manager_name
+  manager_type = var.manager_type
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the permission set resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the workspace ID in which the permission set.
+  Changing this creates a new permission set.
+
+* `name` - (Required, String) Specifies the name of the permission set. The name can contain `1` to `128` characters.
+  Only letters, digits, and underscores (_) are allowed.
+
+* `parent_id` - (Required, String, ForceNew) Specifies the parent ID of the permission set.
```

Comment:
1. why it is required because it is just optional in API;
2. what's the value of `parent_id` if we want to create a root permission set?

Reply:
The API document is wrong, parent_id is a required parameter